Dockerfile.ocp

```
@@ -34,6 +34,9 @@ RUN set -euxo pipefail && \
 # build final image
 FROM registry.ci.openshift.org/ocp/5.0:base-rhel9
 
+# Add PQC support until we have pqc enabled ubi minimal images available
+update-crypto-policies --set DEFAULT:PQ
```

Hadolint (2.14.0)

[error] unexpected 'u'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)